tools/document-pii/src/pii_finder.rs

```diff
@@ -21,6 +21,29 @@ pub struct TypeAndField {
     pub field_ident: String,
 }
 
+#[derive(Ord, PartialOrd, PartialEq, Eq, Hash, Clone, Debug, Default)]
+pub struct FieldsWithAttribute {
+    pub type_and_fields: Vec<TypeAndField>,
+    pub attributes: BTreeMap<String, Option<String>>,
```

Comment:
attribute are string -> Option<String> because some attributes are key-value pair while others just have the key. e.g `pii = "true"` vs `additional_properties`